### ISSUE TITLE
Clustering markers even when a single exists, the default way not like SingleMarkerMode

### ISSUE THREAD

**Issue by coyotemrm** [Leaflet/Leaflet.markercluster#801]
Hi,
I am trying to change the clustering minimum threshold from greater than 1 markers to "greater and equal to 1" marker. Basically, I want to have any single marker on the map to show as a cluster of one, just like what we see on single marker mode. But I need to have the markers with spiderfy for all clusters with their original icons since I am using different logos for each marker, just like when the single marker mode is off.  
To summarize, I need to have the single marker mode off but have the single markers act as a cluster of one.
![image](https://user-images.githubusercontent.com/16028743/27514834-a08cdefc-59aa-11e7-874c-0b25599378e6.png)
I would be grateful if you could guide me what to do.
Thanks.
M.

**Comment by danzel** [comment]
Sorry not supported. Would consider a PR to add it, but hm yeah tricky one.

**Comment by ivanakimov** [comment]
This would be super useful to specify minimum number of markers to cluster.

**Comment by kirantpatil** [comment]
Hi,

Any updates on this issue ?

I am also in need of this functionality.

**Comment by rnbwspctrm** [comment]
+1 I am also in need of this.
